### PR TITLE
Implemented django-tables2 to lobby list view

### LINKED ISSUE #128: Implement django-tables2 in lobby list view
**Goal:** We currently only list the available lobbies in bullet points. To enhance use experience and to be able to implement filtering capabilities in the future, we want to implement `django-tables2` to create nicer tables in the lobby list view.

**Deliverables:**
- PR that creates `tables.py` and makes appropriate changes to the `views.py` classes and respective HTML files

### PR DESCRIPTION
**Pull request by jiyeongchloe**
To implement django-tables2 to lobby list view, I
- added `django_tables2` to requirements/base.txt
- changed the lobby-list definitions and classes in `urls.py` and `views.py`
- added `tables.py`
- modified `lobby-list.html` to render the table instead of listing the lobbies in bullets

### CODE REVIEW

**Review by nugerhard — APPROVED** [review]
Works on my end. Maybe some thought needs to be given to what columns are eventually displayed by this lobbies table, but I don't think that's super necessary to figure out right now. Thanks for adding this!

File: requirements/base.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ django-environ==0.10.0  # https://github.com/joke2k/django-environ
 django-allauth==0.57.0  # https://github.com/pennersr/django-allauth
 django-crispy-forms==2.0  # https://github.com/django-crispy-forms/django-crispy-forms
 crispy-bootstrap5==0.7  # https://github.com/django-crispy-forms/crispy-bootstrap5
+django-tables2
```
Comment (elizabethli31) [inline requirements/base.txt]: Remove this - you only need it in local.txt

**Review by elizabethli31 — CHANGES_REQUESTED** [review]
Could you guys update with dev and then I'll review? Because of some changes in the game model in dev, I can't make game objects anymore with the version you guys currently have. Looks fine generally!

File: requirements/base.txt (hunk 1 above)
Comment (jiyeongchloe) [reply requirements/base.txt]: has been removed

**Review by elizabethli31 — APPROVED** [review]
LGTM! This view looks really good and is much cleaner than before. Great work Chloe!

**Comment by jiyeongchloe** [comment]
**Closing Comment:**
PR is merged and closed.

Chloe: added django_tables2 to requirements/base.txt, changed the lobby-list definitions and classes in urls.py and views.py, added tables.py, modified lobby-list.html to render the table instead of listing the lobbies in bullets
Lucien: helped debug issues and migrations problems
Nat: reviewer

**Comment by elizabethli31** [comment]
Issue Score: Excellent

Comments:
Nice improvement from the rudimentary list view that was previously implemented.

**Pull request merged**